core/src/main/scala/runtime/metrics.scala

```diff
@@ -72,10 +69,10 @@ object implicits {
       }
 
       def streamMetrics: M[Stream[Task, WebSocketFrame]] = {
-        val stream: Stream[Task, Text] = time
+        val stream: Stream[Task, Binary] = time
           .awakeEvery[Task](1.second)
           .map { d =>
-            Text(randomMetrics.map(_.toString()) mkString ("\n"))
+            Binary(ListMetrics(randomMetrics.map(_.toProto)).toProto.toByteArray)
```

Comment:
In the next PRs, we should model and accumulate errors related to potential failures.

Reply:
Perfect. At this moment, we don't need accumulate errors because `randomMetrics` always is going to work fine. In the next steps, I will keep that in mind.

Thanks for the reminder.